data/2mass_j22383372−1517573.json

```diff
@@ -0,0 +1,38 @@
+{
+    "Sources": [
+        {
+            "source": "2MASS J22383372-1517573",
+            "ra": 339.64,
+            "dec": -15.2999,
+            "epoch": 2000.0,
+            "equinox": "J2000",
+            "shortname": null,
+            "reference": "Lipp52",
+            "other_references": null,
+            "comments": null
+        }
+    ],
+    "Names": [
+        {
+            "other_name": "2MASS J22383372-1517573"
+        }
+    ],
+    "Spectra": [
+        {
+            "spectrum": "https://bdnyc.s3.amazonaws.com/IRS/2238-1517_IRS_spectrum.fits",
+            "original_spectrum": "https://bdnyc.s3.amazonaws.com/IRS/2238-1517_IRS_spectrum.dat",
+            "local_spectrum": null,
+            "regime": "mir",
+            "telescope": "Spitzer",
+            "instrument": "IRS",
+            "mode": "SL",
+            "observation_date": "2004-06-07T00:00:00",
+            "wavelength_units": "um",
+            "flux_units": "Jy",
+            "wavelength_order": null,
+            "comments": "SL2,SL1,LL2,LL1",
+            "reference": "Suar22",
+            "other_references": null
+        }
+    ]
+}
```

Comment:
I don't think this source should be ingested. It's an M5 and out of scope for SIMPLE.